core/components/atoms/message/__stories__/variants/Appearance.story.jsx

```diff
@@ -1,39 +1,93 @@
 import * as React from 'react';
-import { Message, Text } from '@/index';
+import { Message, Text, Row, Column } from '@/index';
 
 // CSF format story
 export const appearanceWithoutTitle = () => {
-  const appearances = ['alert', 'info', 'success', 'warning'];
-  const innerStyle = {
-    display: 'flex',
-    'align-items': 'center',
-    'flex-direction': 'column',
-  };
   return (
-    <div className="Row">
-      {appearances.map((appear, ind) => {
-        return (
-          <div key={ind} style={innerStyle} className="mr-7 mb-7 w-25 d-flex">
-            <Message
-              appearance={appear}
-              description="Patient record has been updated with new records."
-              actions={
-                <>
-                  <Text className="cursor-pointer" appearance="link">
-                    Action 1
-                  </Text>
-                  <Text className="ml-5 cursor-pointer" appearance="link">
-                    Action 2
-                  </Text>
-                </>
-              }
-            />
-            <br />
-            <Text weight="strong">{appear.charAt(0).toUpperCase() + appear.slice(1)}</Text>
-          </div>
-        );
-      })}
-    </div>
+    <Row>
```

Comment:
Please keep the mapping part as it will be readable as the source code and example output will be still the same.  Please use this sandbox as example of how examples can be improved https://codesandbox.io/s/trusting-https-ny9i1d?file=/index.js , Please don't copy the sandbox example in story.